src/projects/republic/config/englishRepublicLabels.ts

```diff
@@ -34,4 +34,14 @@ export const englishRepublicLabels = Object.assign({}, englishLabels, {
 
   // text panel titles:
   self: "Text",
+
+  COM: "commission",
```

Comment:
Let's ask Republic how they want to translate the entities.

Reply:
Labels can be changed easily: can be done after the PR is merged? To prevent additional fixing and testing merge conflicts, etc? (This is quite a big PR)